docs/reference/query-dsl/semantic-query.asciidoc

```diff
@@ -0,0 +1,177 @@
+
+field::
+(Required, string)
+The `semantic_text` field to perform the query on.
+
+query::
+(Required, string)
+The query text to be searched for on the field.
+
+
+Refer to <<semantic-search-semantic-text,this tutorial>> to learn more about semantic search using `semantic_text` and `semantic` query.
+
+[discrete]
+[[hybrid-search-semantic]]
+==== Hybrid search with the `semantic` query
+
+The `semantic` query can be used as a part of a hybrid search where the `semantic` query is combined with lexical queries.
+For example, the query below finds documents with title matching "mountain lake", and combines them with results from a semantic search on the field `title_semantic`, that is a `semantic_text` field.
+The combined documents are then scored, and the top 3 top scored documents are returned.
+
+[source,console]
+------------------------------------------------------------
+POST my-index/_search
+{
+  "size" : 3,
+  "query": {
+    "bool": {
+      "should": [
+        {
+          "match": {
+            "title": {
+              "query": "mountain lake",
+              "boost": 1
+            }
+          }
+        },
+        {
+          "semantic": {
+            "field": "title_semantic",
+            "query": "mountain lake",
+            "boost": 2
+          }
+        }
+      ]
+    }
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+
+You can also use semantic_text as part of <<rrf,Reciprocal Rank Fusion>>:
+
+[source,console]
+------------------------------------------------------------
+GET my-index/_search
+{
+  "retriever": {
+    "rrf": {
+      "retrievers": [
+        {
+          "standard": {
+            "query": {
+              "term": {
+                "text": "shoes"
+              }
+            }
+          }
+        },
+        {
+          "semantic": {
+            "field": "semantic_field",
+            "query": "shoes"
+          }
+        }
+      ],
+      "rank_window_size": 50,
+      "rank_constant": 20
+    }
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+
+
+[discrete]
+[[search-sparse-inference]]
+==== Search with `sparse_embedding` inference
+
+When the {infer} endpoint uses a `sparse_embedding` model, you can use a `sparse_vector` query on a <<semantic-text,`semantic_text`>> field in the following way:
+
+[source,console]
+------------------------------------------------------------
+GET test-index/_search
+{
+  "query": {
+    "nested": {
+      "path": "inference_field.inference.chunks",
+      "query": {
+        "sparse_vector": {
+          "field": "inference_field.inference.chunks.embeddings",
+          "inference_id": "my-inference-id",
+          "query": "mountain lake"
+        }
+      }
+    }
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+
+You can customize the `sparse_vector` query to include specific settings, like pruning configuration.
```

Comment:
Maybe link to token pruning section of sparse vector docs here?

Reply:
I linked to the parameters section as the pruning section doesn't have an anchor.